sql/aiven_extras.sql

```diff
@@ -496,6 +496,22 @@ END;
 $$;
 
 
+DROP FUNCTION IF EXISTS aiven_extras.explain_statement(TEXT);
+CREATE FUNCTION aiven_extras.explain_statement(
+    arg_query TEXT,
+    OUT execution_plan JSON
+)
+RETURNS SETOF JSON
+RETURNS NULL ON NULL INPUT
+LANGUAGE plpgsql
+SECURITY DEFINER
```

Comment:
search path is missing.

Reply:
Search path can't be set here, see commit message.